packages/mds-db/events.ts

```diff
@@ -114,6 +114,83 @@ export async function readEvents(params: ReadEventsQueryParams): Promise<ReadEve
   }
 }
 
+// -000
+
+export interface TripEvents {
+  [trip_id: string]: VehicleEvent[]
+}
+
+export interface TripEventsResult {
+  trips: TripEvents
+  count: number
```

Comment:
Can we rename this to something like `eventCount` so it's more clear? I thought at first that this could be the # of trips or total # of events.

Reply:
thanks!